config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade-addons-before-controlplane.yaml

```diff
@@ -0,0 +1,46 @@
+# AUTOGENERATED by https://git.k8s.io/kubeadm/kinder/ci/tools/update-workflows
+periodics:
+
+- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-addons-before-controlplane-1-27-latest
+  cluster: k8s-infra-prow-build
+  interval: 2h
```

Comment:
Not sure if the interval is too short.
24h or 12h would be enough for this feature?

Reply:
it's common for our "latest" test jobs

Reply:
Then that is fine.
- The current CI triggering is not that smart to do some back-off strategy.

Reply:
yes, we can spread out latest jobs less often in the generator e.g. 4 hours at some future release